src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,36 @@
 import React from 'react';
+import { SortingType } from '../../types/sortingType';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as filterActions } from '../../features/filter';
 
 export const TodoFilter: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const { status, query } = useAppSelector((state) => state.filter);
+
+  const handleFilterChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    dispatch(filterActions.setStatus(event.target.value as SortingType));
+  };
+
+  const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    dispatch(filterActions.setQuery(event.target.value));
+  };
+
+  const handleClearSerch = () => {
```

Comment:
There is a typo in the function name `handleClearSerch`. It should be `handleClearSearch`.